todo_app/todo_app/settings.py

```diff
@@ -123,3 +133,7 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:3000',
+    "http://127.0.0.1:3000"]
```

Comment:
You shouldn't need both of these. Also, stick with single quotes on the backend.